public/docs/ts/latest/guide/reactive-forms.jade

```diff
@@ -0,0 +1,879 @@
+  tracks the value and validity state of a numerically indexed _array_ of `AbstractControl` instances.
+
+  You'll learn more about these classes as you build your first reactive form.
+
+.l-main-section
+a#setup
+:marked
+  ## Setup
+
+  Follow the steps in the [_Setup_ guide](../setup.html "Setup guide") 
+  for creating a new project folder (perhaps called `reactive-forms`)
+  based on the _QuickStart seed_.
+
+.l-main-section
+a#data-model
+:marked
+  ## Create a data model
+  The focus of this guide is a reactive forms component that edits a hero.
+  You'll need a `hero` class and some hero data.
+  Create a new `data-model.ts` file in the `app` directory and copy the content below into it.
+
++makeExample('reactive-forms/ts/app/data-model.ts', '','app/data-model.ts')(format=".")
+
+:marked
+  The file exports two classes and two constants. The `Address` 
+  and `Hero` classes define the application _data model_. 
+  The `heroes` and `states` constants supply the test data. 
+
+.l-main-section
+a#create-component
+:marked
+  ## Create a _reactive forms_ component
+  Make a new file called 
+  `hero-detail.component.ts` in the `app` directory and import these symbols:
+
++makeExample('reactive-forms/ts/app/hero-detail.component.ts', 'reactive-comp-imports','app/hero-detail.component.ts')(format=".")
+
+:marked
+  Here’s a summary description of the imported symbols, whose meaning and use
+  will become clear over the course of this guide:
+
+  - `Component`: the Angular decorator that describes the component in metadata.
+  - `EventEmitter`: a class that emits events that are handled by a parent component.
+  - `Input`: identifies a component's public input properties for data binding by a parent component.
+  - `Output`: identifies a component's public output properties for data binding by a parent component.  
+  - `OnChanges`: a component lifecycle hook that Angular calls when input values change.
+  - `FormArray`: an array of `FormControls`.
+  - `FormBuilder`: a builder of `FormControls`, `FormGroups`, and `FormArrays`.
+  - `FormControl`: a representation of an HTML form control.
+  - `FormGroup`: a group of `FormControls`, `FormArrays`, and other `FormGroups`.
+  - `Address` and `Hero` are application model classes.
+  - `states`: U.S. states (test data). 
+
+  Now enter the `@Component` decorator that specifies the `HeroDetailComponent` metadata:
+
++makeExample('reactive-forms/ts/app/hero-detail.component.ts', 'reactive-comp-metadata','app/hero-detail.component.ts (excerpt)')(format=".")
+
+:marked
+  The `moduleId: module.id` lets you use 
+  [component-relative paths](./cookbook/component-relative-paths.html) in file URLs. 
+  The `selector` tells Angular the name of the element tag that renders this form. 
+  The `templateUrl` is the location of the HTML template that displays the form.
+
+  Next, create an exported `HeroDetailComponent` class with a `FormControl`. 
```

Comment:
Should this be "FormGroup" here and not "FormControl"? (I'm reading these in github so am not seeing the code examples.)